stories/Drag.stories.tsx

```diff
@@ -479,7 +479,18 @@ export const NestedNodeRearranging = () => {
       <Canvas
         nodes={nodes}
         edges={edges}
-        node={<Node dragType="node" />}
+        node={(node: NodeProps) => {
+          // Prevent parent nodes with large number of children from dragging, but allow children to drag
```

Comment:
I don't think we want to add this arbitrary limitation.

Reply:
this was just to demonstrate that explicit child properties override their parent properties per the referenced issue